### PR TITLE
[KHRGA-92] Updated DT math array types

### PR DESCRIPTION
**Pull request by imvucic**
Issue: KHRGA-92

### CODE REVIEW

File: source/iface/math-array-types.rst [hunk 1]
```diff
@@ -7,3 +7,564 @@
 ****************
 Math array types
 ****************
+
```
Comment (mkinsner) [inline source/iface/math-array-types.rst:10]: Other sections have a link/reference to the SYCL spec.  Can that be added here?

File: source/iface/math-array-types.rst [hunk 2]
```diff
@@ -7,3 +7,564 @@
...
+``begin``
+=========
+
+::
+
+  iterator begin();
+
+Returns an iterator referring to the first element stored within the
+``v marray``.
```
Comment (mkinsner) [inline source/iface/math-array-types.rst:162]:
```suggestion
``marray``.
```

Comment (mkinsner) [reply source/iface/math-array-types.rst:162]: https://github.com/KhronosGroup/SYCL-Docs/pull/523

File: source/iface/math-array-types.rst [hunk 3]
```diff
@@ -7,3 +7,564 @@
...
+
+::
+
+  const_iterator begin() const
+
+Returns a ``const`` iterator referring to the first element stored within the
+``v marray``.
```
Comment (mkinsner) [inline source/iface/math-array-types.rst:169]:
```suggestion
``marray``.
```

File: source/iface/math-array-types.rst [hunk 4]
```diff
@@ -7,3 +7,564 @@
...
+
+``end``
+=======
+
+::
+
+  iterator begin();
+
+Returns an iterator referring to the one past the last element stored
+within the ``v marray``.
```
Comment (mkinsner) [inline source/iface/math-array-types.rst:179]:
```suggestion
within the ``marray``.
```

File: source/iface/math-array-types.rst [hunk 5]
```diff
@@ -7,3 +7,564 @@
...
+
+::
+
+  const_iterator end() const
+
+Returns a ``const`` iterator referring to the one past the last element
+stored within the ``v marray``.
```
Comment (mkinsner) [inline source/iface/math-array-types.rst:186]:
```suggestion
stored within the ``marray``.
```

File: source/iface/math-array-types.rst (hunk 1 above)
Comment (imvucic) [reply source/iface/math-array-types.rst:10]: Added with https://github.com/KhronosGroup/SYCL_Reference/pull/145/commits/38d5460d79e12d1e98008c14f7f6a2bc6550a6d5

File: source/iface/math-array-types.rst (hunk 2 above)
Comment (imvucic) [reply source/iface/math-array-types.rst]: Fixed with : https://github.com/KhronosGroup/SYCL_Reference/pull/145/commits/38d5460d79e12d1e98008c14f7f6a2bc6550a6d5

File: source/iface/math-array-types.rst (hunk 3 above)
Comment (imvucic) [reply source/iface/math-array-types.rst]: Fixed with : https://github.com/KhronosGroup/SYCL_Reference/pull/145/commits/38d5460d79e12d1e98008c14f7f6a2bc6550a6d5

File: source/iface/math-array-types.rst (hunk 4 above)
Comment (imvucic) [reply source/iface/math-array-types.rst]: Fixed with : https://github.com/KhronosGroup/SYCL_Reference/pull/145/commits/38d5460d79e12d1e98008c14f7f6a2bc6550a6d5

File: source/iface/math-array-types.rst (hunk 5 above)
Comment (imvucic) [reply source/iface/math-array-types.rst]: Fixed with : https://github.com/KhronosGroup/SYCL_Reference/pull/145/commits/38d5460d79e12d1e98008c14f7f6a2bc6550a6d5

**Review by mkinsner — APPROVED** [review]

**Pull request merged**